components/dashboard/src/teams/TeamNetworking.tsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { isGitpodIo } from "../utils";
+import React from "react";
+import { Heading2, Heading3, Subheading } from "../components/typography/headings";
+import { OrgSettingsPage } from "./OrgSettingsPage";
+import { ConfigurationSettingsField } from "../repositories/detail/ConfigurationSettingsField";
+import { useDocumentTitle } from "../hooks/use-document-title";
+import { LinkButton } from "@podkit/buttons/LinkButton";
+import { CheckCircle2Icon } from "lucide-react";
+import { Redirect } from "react-router";
+import PillLabel from "../components/PillLabel";
+
+export default function TeamPoliciesPage() {
+    useDocumentTitle("Organization Settings - Networking");
+
+    if (!isGitpodIo) {
+        return <Redirect to="/settings" />;
+    }
+
+    return (
+        <>
+            <OrgSettingsPage>
+                <div className="space-y-8">
+                    <div>
+                        <Heading2 className="flex items-center gap-4">
+                            Networking
+                            <PillLabel type="warn" className="!px-3 !py-0.5 !text-xs border">
+                                <span className="text-sm font-semibold text-pk-content-secondary">Enterprise</span>
+                            </PillLabel>
+                        </Heading2>
+                        <Subheading className="mt-1">Manage your organization's networking settings.</Subheading>
+                    </div>
+
+                    <SelfHostedCalloutCard />
+                    <DeployedRegionCard />
+                    <VPNCard />
+                </div>
+            </OrgSettingsPage>
+        </>
+    );
+}
+
+const SelfHostedCalloutCard = () => {
+    return (
+        <ConfigurationSettingsField className="bg-pk-surface-secondary">
```

Comment:
The borders between the grey info boxes and the white seem to be different (32px vs 16px). When you swap tabs, this mean the layout feels like it's shifting. Is it possible to align those margins? 

|Grey|White|
|-|-|
|![image](https://github.com/gitpod-io/gitpod/assets/5528307/921eadb6-bd5a-4641-8309-6f8f9577c5cf)|![image](https://github.com/gitpod-io/gitpod/assets/5528307/195d562f-6e67-4935-930d-b1c83d083cf0)|

Reply:
ahh! `Collaboration and sharing` component is using outdated + inconsistent design style (un-related to this change) -- I'll fix it.